quickstart/mutators.markdown

```diff
@@ -947,8 +954,303 @@ The switch mutator finds the first label within a switch statement that differs
 
 *Thanks to Stefan Penndorf who contributed this documentation.*
 
-<hr/>
+<a name="ABS" id="EXPERIMENTAL_ABS"></a>
+
+Negation Mutator (ABS)
+-------------------------------------------------
+This mutator replace any use of a numeric variable (local valiable, field, array cell) with its negation. For example:
+```java
+public float get(final float i) {
+  return i;
+}
+```
 
+will be mutated to
+
+```java
+public float get(final float i) {
+  return -i;
+}
+```
+<a name="AOR" id="EXPERIMENTAL_AOR"></a>
+
+Arithmetic Operator Replacement Mutator (AOR)
+-------------------------------------------------
+Like the math mutator, this mutator replaces binary arithmetic operations for either integer or floating-point arithmetic with another operation. The mutator is composed of 4 sub-mutators (AOR_1 to AOR_4) that mutate operators according to the table below.
+<table>
```

Comment:
Hi @LaurentTho3, thanks for the pull request.

We (now) favor markdown tables over HTML tables in the site sources because they make the sources more readable. I guess you did not noticed it because your work is based on quite an old version of the site.

Could you change the HTML tables to markdown tables ? (be mindful of the `{:.table }` thing that we add after every table, it's not part of _vanilla_ markdown but it's important for the site style) 

You might also want to rebase you work to a more recent version of the site (since github don't detect any merge conflict I guess you don't _have_ do do it, but it would avoid the right of strange merge result, and you would get sample of the new tables in you workspace)

Reply:
Hi @maxgabut,

I think it should be okay now. Let me know if there is anything else I should fix.

Reply:
Thanks for updating this PR. Some last tiny details caught my eye (see other comments), but it's really simple and quick to do.